public/game/view-models/app-viewmodel.coffee

```diff
@@ -59,7 +60,7 @@ class AppViewModel
 		@occurrenceIndicator.string['<-'] @selector
 
 		@userPanelViewModel = new UserPanelViewModel @user_data
-		@roundTimerViewModel = new TimespanViewModel @countdown, dateTimeFormats['m:ss']
+		@roundTimerViewModel = new CountdownCircleViewModel @countdown, 2 * 60 * 1000, dateTimeFormats['m:ss'], { radius: 40, strokeWidth: 5 }
```

Comment:
remind me, what have we decided with sharing round duration between client and server?